_config/config.yml

```diff
@@ -6,3 +6,7 @@ SilverStripe\Admin\LeftAndMain:
     - 'silverstripe/security-extensions: client/dist/js/bundle.js'
   extra_requirements_css:
     - 'silverstripe/security-extensions: client/dist/styles/bundle.css'
+
+SilverStripe\Security\Member:
```

Comment:
In #1 I've created an `extensions.yml` file. Once both merged we can move this into that file

Reply:
relocated